README.md

```diff
@@ -26,7 +26,7 @@ Check [Documentation](https://specmatic.in/documentation.html) for more informat
 
 ## Test helper library
 
-`import { startStubServer, startTestServer, loadDynamicStub } from 'specmatic';`
+`import { startStubServer, startTestServer, loadDynamicStub, installSpecmatics } from 'specmatic';`
```

Comment:
can we rename this to installContracts?
This is too confusing with `npm install specmatic` the command to install specmatic node module

Reply:
done.